libsmtutil/SMTPortfolio.h

```diff
@@ -62,12 +63,19 @@ class SMTPortfolio: public SolverInterface
 
 	std::vector<std::string> unhandledQueries() override;
 	size_t solvers() override { return m_solvers.size(); }
+
+	std::string getSMTLIBCode(std::vector<Expression> const& _expressionsToEvaluate);
```

Comment:
Shouldn't this function be in `SolverInterface` too with `override`s here and in the other solver interfaces?

Reply:
@leonardoalt I would rather leave it like it is. We'll make this functionality specific for SMTLib2 solvers.